docs/resources/aws_s3_bucket_object.md

```diff
@@ -0,0 +1,76 @@
+---
+title: About the aws_s3_bucket_object Resource
+---
+
+# aws_iam_users
+
+Use the `aws_s3_bucket_object` InSpec audit resource to test properties of a s3 bucket object(file).
```

Comment:
I'd prefer not to use the `object(file)`  phrasing.  If you'd like to explain it the first time, that seems fine, but we should use standard AWS notation elsewhere.